### ISSUE TITLE
It is extremely slow and severely impacts usability, taking ten times longer compared to using other people's scripts

### ISSUE THREAD

**Issue by jxhxgt** [bmaltais/kohya_ss#1707]
steps:   0%|                                                                                  | 0/7400 [00:00<?, ?it/s]
epoch 1/100
E:\kohya_ss\venv\lib\site-packages\torch\utils\checkpoint.py:31: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")
steps:   0%|                                                       | 6/7400 [03:50<79:02:37, 38.48s/it, avr_loss=0.148]

**Comment by jxhxgt** [comment]
03:08:52-530820 INFO     Version: v22.2.1

03:08:52-535884 INFO     nVidia toolkit detected
03:08:53-795726 INFO     Torch 2.0.1+cu118
03:08:53-826639 INFO     Torch backend: nVidia CUDA 11.8 cuDNN 8700
03:08:53-828594 INFO     Torch detected GPU: NVIDIA GeForce RTX 4070 VRAM 12282 Arch (8, 9) Cores 46
03:08:53-830104 INFO     Verifying modules installation status from requirements_windows_torch2.txt...
03:08:53-833630 INFO     Verifying modules installation status from requirements.txt...
03:08:56-028754 INFO     headless: False
03:08:56-031727 INFO     Load CSS...
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
03:09:03-156657 INFO     Start training Dreambooth...
03:09:03-159563 INFO     Valid image folder names found in: E:/xlzl/tupian/jpg
03:09:03-162493 INFO     Folder 1_sakichan' style : steps 439
03:09:03-163502 INFO     max_train_steps (439 / 8 / 1 * 50 * 1) = 2744
03:09:03-164514 INFO     stop_text_encoder_training = -1
03:09:03-164514 INFO     lr_warmup_steps = 274
03:09:03-165488 INFO     Saving training config to
                         E:/sd-webui-aki/sd-webui-aki-v4.2/models/Stable-diffusion\last_20231125-030903.json...
03:09:03-167437 INFO     accelerate launch --num_cpu_threads_per_process=20 "./train_db.py" --enable_bucket
                         --min_bucket_reso=256 --max_bucket_reso=2048
                         --pretrained_model_name_or_path="E:\sd-webui-aki\sd-webui-aki-v4.2\models\Stable-diffusion\DB.s
                         afetensors" --train_data_dir="E:/xlzl/tupian/jpg" --resolution="1024,1024"
                         --output_dir="E:/sd-webui-aki/sd-webui-aki-v4.2/models/Stable-diffusion"
                         --stop_text_encoder_training=-1 --save_model_as=safetensors --output_name="last"
                         --lr_scheduler_num_cycles="4" --lr_scheduler_power="5" --max_token_length=225
                         --max_train_epochs="100" --max_data_loader_n_workers="20" --learning_rate_te="0.0"
                         --learning_rate="8e-06" --lr_scheduler="polynomial" --lr_warmup_steps="274"

**Comment by nokeyfan** [comment]
Same here, Version v22.2.0 and v22.2.1 are very slow, tested with older config settings. On RTX 2070 it went from 1.80 it/s to about 40 it/s. I sticked to 22.1.1 that's the speed is fast. And the presets on v22.2.0 and v22.2.1 not working (it was posted on other post, but worth mention it.)
